src/styles/catalog.scss

```diff
@@ -0,0 +1,44 @@
+.main {
+  padding: 50px 40px;
+}
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(100px, 1fr));
+  row-gap: 48px;
+  column-gap: 48px;
+  max-width: 1200px;
+  margin: 0 auto;
+  place-items: center;
+}
+
+.card {
+  margin-bottom: -2px;
+}
```

Comment:
Here is something wrong. And don't add margins to BEM block, and styles for `card` should be in card styles file